pkgs/by-name/sp/spotube/package.nix

```diff
@@ -121,5 +126,26 @@ let
           --prefix PATH : ${lib.makeBinPath [ xdg-user-dirs ]}
     '';
   };
+
+  meta = with lib; {
```

Comment:
Could you revert the changes made to `meta` (other than adding `aarch64-linux to platforms)?
`with lib;` is not very encouraged nowadays.

Reply:
```
meta = {
    description = "Open source, cross-platform Spotify client compatible across multiple platforms";
    longDescription = ''
      Spotube is an open source, cross-platform Spotify client compatible across
      multiple platforms utilizing Spotify's data API and YouTube (or Piped.video or JioSaavn)
      as an audio source, eliminating the need for Spotify Premium
    '';
    downloadPage = "https://github.com/KRTirtho/spotube/releases";
    homepage = "https://spotube.krtirtho.dev/";
    license = lib.licenses.bsdOriginal;
    mainProgram = "spotube";
    maintainers = with lib.maintainers; [ tomasajt ];
    platforms = [
      "x86_64-linux"
      "x86_64-darwin"
      "aarch64-darwin"
      "aarch64-linux"
    ];
    sourceProvenance = with lib.sourceTypes; [ binaryNativeCode ];
  };
```

okay?

Reply:
This PR does not cleanly list package outputs after merging. do you know why this err?

Reply:
> okay?

it should be the same as before this PR (except the platforms change)


> This PR does not cleanly list package outputs after merging. do you know why this err?

Your first commit title is not following the commit convention, and ofborg can't parse that.

Also, please squash your `spotube:` related commits into one commit.
*One way to do it is using: `git rebase -i HEAD~4`*

Reply:
I have changed direct on GitHub, im gonna do it by git now, the title can be spotube: 3.8.3 → 3.9.0 | add aarch64-linux support or other way?

Reply:
> I have changed direct on GitHub, im gonna do it by git now, the title can be spotube: 3.8.3 → 3.9.0 | add aarch64-linux support?

I think it's fine. Though I am not 100% sure about using |
 I usually use ,